graphics/ttfautohint/Portfile

```diff
@@ -17,8 +17,8 @@ maintainers        @tuffnatty openmaintainer
 license            {freetype GPL-2}
 master_sites       sourceforge:project/freetype/ttfautohint/${version}
 
-checksums          rmd160 74697057f5d27afb112b7b61dcc4b286fb3a9b26 \
-                   sha256 386741701596a8b2d5fb744901922ed2bd740490f7e6c81e5d7e83ac677889a7
+checksums          rmd160 67e59778818bd0c71e4eff6ca437bd994800797e \
+                   sha256 87bb4932571ad57536a7cc20b31fd15bc68cb5429977eb43d903fa61617cf87e
```

Comment:
We are now adding the `size` parameter to `checksums`.

Reply:
Done.